test/server/routes/users.test.js

```diff
@@ -145,12 +144,11 @@ describe('server/routes/users', () => {
       const encryptedToken = crypto[CIPHER].encrypt(secret.base32, SECRET_KEY).toString();
       const user = await models.User.create({ email: 'mopsa@mopsa.mopsa', twoFactorAuthToken: encryptedToken });
       const currentToken = user.jwt();
-      const twoFactorAuthenticatorCode = new TOTP({
+      const twoFactorAuthenticatorCode = speakeasy.totp({
         algorithm: 'SHA1',
-        digits: 6,
-        period: 30,
-        secret: Secret.fromB32(secret.base32),
-      }).generate();
+        encoding: 'base32',
+        secret: secret.base32,
```

Comment:
Should we use `window` here?

Reply:
It isn't a param for `speakeasy.totp`. It's a param on `totp.verify` to allow a bit of wiggle room when you're decrypting. The thing you set here on creating the TOTP would be 'period' which specifies how big the window is. But you don't set the amount of windows you can have on decryption.

Reply:
Ok, make sense.

It's not clear however using "decrypting" and "decryption" in this context (unless I'm wrong). We're just generating a token (and later verifying it), there is no encryption in the process (out of the DB encryption which is something separate).